src/main/java/com/czertainly/core/messaging/listeners/NotificationListener.java

```diff
@@ -69,12 +139,82 @@ public void processMessage(NotificationMessage notificationMessage) {
                                         data.getApprovalUuid(), data.getObjectUuid(), data.getCreatorUsername(), data.getStatus().getLabel()),
                                 getApprovalNotificationDetail(data),
                                 notificationMessage);
+                        if (notificationTypeStringMap != null) {
+                            String notificationInstanceUUID = notificationTypeStringMap.get(NotificationType.APPROVAL_CLOSED);
+                            if (notificationInstanceUUID != null) {
+                                sendExternalNotifications(notificationInstanceUUID, notificationMessage, NotificationType.APPROVAL_CLOSED);
+                            }
+                        }
                     }
                 }
             }
         }
     }
 
+
+    private void sendExternalNotifications(String notificationInstanceUUID, NotificationMessage notificationMessage, NotificationType notificationType) {
```

Comment:
I would suggest to use `UUID` instead of `String` for `notificationInstanceUuid`

Reply:
changed it